storybook/cypress/e2e/panel.cy.ts

```diff
@@ -9,7 +9,16 @@ describe("Panel", () => {
   });
 
   it("should matchImageSnapshot", () => {
-    cy.get("dso-panel.hydrated").matchImageSnapshot();
+    /**
+     * Wait for all fonts to load before taking the screenshot.
+     * See #2748 for more information.
+     */
+    cy.document()
+      .its("fonts.status")
+      .should("equal", "loaded")
+      .then(() => {
+        cy.get("dso-panel.hydrated").matchImageSnapshot();
+      });
```

Comment:
Ik denk niet dat je die `.then()` nodig hebt en dat je een nieuwe chain met `.get("dso-panel.hydrated")` kan starten.

Ik wil dit graag doorhebben omdat we klaar zijn met die flaky test. Dus laten we hier op een later moment eens 'n betere fix voor bedenken. Ik ben blij wat we hier doen, maar het schaalt natuurlijk niet: Bij de volgende italic styling lopen we weer hetzelfde risico.

Wat wel interessant is, waarom faalt altijd de Ozon Content test...

Reply:
De renvooi pagina van ozon-content is groot/veel/zwaar met onderin 2 grote platen met daarboven ivm de renvooi italic text.... dus is de default-timeout van 4000ms niet voldoende om in de overwrite van `screenshot` de voorwaarde `.its("fonts.status").should("equal", "loaded")` te resolven.

Ik heb nu een andere insteek gekozen, door de in de renvooi test in `ozon-content.cy.ts` de default timeout te overrriden.

Reply:
De Ci/CD build is <span style="color:#39870c">groen</span> https://github.com/dso-toolkit/dso-toolkit/actions/runs/11385194967/job/31674505123

Reply:
Maar kunnen we dan niet beter een kleinere plaat inchecken? Dat ding is bijna 4MB

Reply:
Ja, dat kan natuurlijk ook. Ga ik doen 👍

Reply:
Ik heb de afbeelding aanzienlijk verkleind. Helaas is dezelfde fout nog steeds op. Ik denk dat mijn volgende stap is om die afbeelding er eens helemaal uit te halen. Kijken wat er dan gebeurt. Misschien heeft het wel helemaal niets met (de grote van) de afbeelding te maken.